### PR TITLE
Fix #130

### LINKED ISSUE #130: Fix the =-Space Expression Bug
At the Moment the ALGExpressionType>>getCommand Method will cut the first two characters from the input and run the rest. That means that only "= [EXPRESSION]" will work not "=[EXPRESSION]".
This should be an quite easy fix we should do before submission.

Remember:
- [x] Write a regression test

### PR DESCRIPTION
**Pull request by kej-jay**

### CODE REVIEW

**Review by BenBals — APPROVED** [review]
Looks good to me.

File: packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st [hunk 1]
```diff
@@ -1,6 +1,9 @@
 actions
 getCommand
-	| textFieldContents |
+	| textFieldContents startIndex |
 	textFieldContents := self typeObject topmostQuery.
-
-	^ textFieldContents copyFrom: 3 to: textFieldContents size"first 2 characters are do or '= '"
\ No newline at end of file
+	(textFieldContents at: 1) == $=
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st]: Just some nitpicking:
`textFieldContents first = $=`
 would be a bit more squeak-like

**Review by sinnaj-r — APPROVED** [review]
Looks good 👍 
Just got one small comment, after that you should be fine to merge

File: packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st (hunk 1 above)
Comment (kej-jay) [reply packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st]: did that :+1:

Comment (sinnaj-r) [reply packages/Algernon.package/ALGExpressionType.class/instance/getCommand.st]: Thank you 👍

**Comment by kej-jay** [comment]
Will merge this now!

**Pull request merged**